tracker/data/notes/anypercent/encounters_notes.csv

```diff
@@ -1,117 +0,0 @@
-#name,type,initiative,label,min,default,max
```

Comment:
Do we know why all these files were included to begin with? I'm hesitant to check-mark removing 100 files without at least a little bit of context.

Reply:
They were included cause the whole RNG Tracker repository that the RTA community uses was just dropped into the "tracker" folder. That includes all of its UI code and the default tracking notes RTA runners use when working with the UI.

Reply:
Can confirm, I just copied the whole thing and treated it as a library for the parts needed within the TAS.